src/node.cc

```diff
@@ -293,10 +293,13 @@ static struct {
                     "so event tracing is not available.\n");
   }
   void StopTracingAgent() {}
+#endif  // !NODE_USE_V8_PLATFORM
+
+#if NODE_USE_V8_PLATFORM == 0 || HAVE_INSPECTOR == 0
```

Comment:
shouldn't it be `#if !defined(NODE_USE_V8_PLATFORM) || !defined(HAVE_INSPECTOR)`?